packages/core-web/src/styles/markbind.css

```diff
@@ -27,8 +27,7 @@ pre.hljs > code {
     background: none;
 }
 
-pre > code.hljs {
```

Comment:
Yup, it violates the `property-no-vendor-prefix` rule.
<img width="723" alt="Screenshot 2024-02-25 at 8 15 11 PM" src="https://github.com/MarkBind/markbind/assets/122192553/ccbbae1a-4d7b-4f56-bc80-aed66f55e779">

We can solve this issue by ignoring the property.
<img width="707" alt="Screenshot 2024-02-25 at 8 28 32 PM" src="https://github.com/MarkBind/markbind/assets/122192553/43a7e426-e173-4e54-af67-5c922ca917dc">

This will keep the original line 
`-webkit-background-clip: padding-box; /* for Safari */`

I think we can do this, what do you think?

Reply:
The understanding I have is that when this code was written, the browsers required prefixing for these due to not supporting the properties unprefixed. Stylelint relies on autoprefixer, an open source project that automatically adds prefixes when needed.

The implication here is that the prefixes are no longer needed due to them being supported by the browser at this point, though they were not supported in the past.

Hence, assuming there is no change in behaviour, I think we should keep the rule and change the CSS.

Reply:
Oh! However do remove the comments "for safari"  / references to other browsers, as these are no longer relevant.

Reply:
Oh I think you are right, for now since browsers do not require vendor prefixes, I think the line `background-clip: padding-box` would be enough to accommodate all browsers.

I have removed the unnecessary comments.